sys/include/walltime.h

```diff
@@ -39,15 +49,15 @@ void walltime_set(struct tm *time);
  *
  * @param[out] time current time output
  * @param[out] ms   current milliseconds output, may be NULL
- *                  resolution depends on the backend
+ *                  resolution depends on the backend, may always be 0
```

Comment:
```suggestion
 *                  always returns 0 if the backend supports no millisecond resolution
```

is that the correct statement?